chore: fix security/issues/112

lucasgonze opened the pull request:
Address tasks in https://github.com/magma/security/issues/112

Signed-off-by: Lucas Gonze <lucas@gonze.com>

<!--
    Tag your PR title with the components that it touches along with
    the type of change
    E.g. "fix(orc8r): Fix reindexer race condition" or "feat(agw) ..."
-->

## Summary

<!-- Enumerate changes you made and why you made them -->

## Test Plan

<!--
    How did you test your change? How do you know it works?
    Add supporting screenshots, terminal pastes, etc. as necessary
-->

## Additional Information

- [ ] This change is backwards-breaking

<!--
    If this is a backwards-breaking change, document the upgrade instructions.
    All upgrade instructions for backwards-breaking changes will be aggregated
    in the next release's changelog so this is very important to fill out.
-->

emakeev reviewed (approved):
LGTM

crasu commented on feg/radius/src/config/samples/radius.tataipx.config.json:
```diff
@@ -23,9 +23,9 @@
                                     "config": {
-                                        "IpxEndpoint": "https://carrieridpoc.mvoipctsi.com:36443/ss7gatewaysvcs",
-                                        "Username": "MapFBtclUser",
-                                        "Password": "X70:-1g$M-Q2^0"
```
This will still be contained in git history - is someone taking care of chaging the credentials of this server/taking it offline?

lucasgonze replied:
Yes, I'm working on it.

Note that this server doesn't belong to us.

lucasgonze committed:
chore: fix security/issues/112

Rebase. Fix merge conflict "CONFLICT (modify/delete): xwf/gateway/docker/docker-compose.yml deleted in HEAD and modified in 161e89fbf0 (chore: fix security/issues/112). Version 161e89fbf0 (chore: fix security/issues/112) of xwf/gateway/docker/docker-compose.yml left in tree."

Signed-off-by: Lucas Gonze <lucas@gonze.com>
